compiler/src/iree/compiler/Codegen/Common/DecomposePackUnPackOps.cpp

```diff
@@ -194,6 +247,28 @@ void DecomposePackUnPackOpsPass::runOnOperation() {
       return signalPassFailure();
     }
   }
+
+  LLVM_DEBUG({
+    llvm::dbgs()
+        << "--- After generalizing tensor.pack and tensor.unpack ops ---\n";
+    funcOp.print(llvm::dbgs(), OpPrintingFlags().useLocalScope());
+    llvm::dbgs() << "\n\n";
+  });
```

Comment:
I'm not a huge fan of these large dumps. They may become problematic if we want to debug and the IR is too large. Would you be able to just dump the ops rewrited by the patterns instead?

Reply:
I think `-debug` can work for the case. My question is -- are we able to dump debug message for a specific pass when we're running many passes? This is useful when we use iree-compile with `debug-only`, debugging pipeline tests, etc. Sometimes we only want debug messages from a pass when we're running many passes. This is the only way to get the information, AFAIK. I could be outdated. I haven't studied this area since a while back.